Packs/QRadar/Integrations/QRadar_v3/QRadar_v3.py

```diff
@@ -1447,6 +1484,74 @@ def validate_long_running_params(params: Dict) -> None:
                                    ' Please set a value for it.')
 
 
+def get_cidrs_indicators(query):
+    """Extracts cidrs from a query"""
+    if not query:
+        return []
+
+    res = demisto.searchIndicators(query=query)
+
+    indicators = []
+    for indicator in res.get('iocs', []):
+        if indicator.get('indicator_type').lower() == 'cidr':
+            indicators.append(indicator.get('value'))
+
+    return indicators
+
+
+def verify_args_for_remote_network_cidr(cidrs_list, cidrs_from_query, name, group, fields):
+    # verify that only one of the arguments is given
+    if cidrs_list and cidrs_from_query:
+        return 'Cannot specify both cidrs and query arguments.'
+
+    # verify that at least one of the arguments is given
+    if not cidrs_list and not cidrs_from_query:
+        return 'Must specify either cidrs or query arguments.'
+
+    # verify that the given cidrs are valid
+    for cidr in cidrs_list:
+        if not re.match(ipv4cidrRegex, cidr) and not re.match(ipv6cidrRegex, cidr):
+            return f'{cidr} is not a valid CIDR.'
+
+    # verify that the given name and group are valid
+    if not NAME_AND_GROUP_REGEX.match(name) or not NAME_AND_GROUP_REGEX.match(group):
+        return 'Name and group arguments only allow letters, numbers, \'_\' and \'-\'.'
+
+    fields_list = argToList(fields)
+    if fields_list:
+        possible_fields = ['id', 'name', 'group', 'cidrs', 'description']
+        for field in fields_list:
+            if field not in possible_fields:
+                return f'{field} is not a valid field. Possible fields are: {possible_fields}.'
+
+
+def is_positive(*values):
+    # checks if all values are positive or None but not a negative number
+    for value in values:
+        if value is not None and value < 1:
```

Comment:
What if value isn't a number?

Reply:
It passes through arg_to_number so that it is either a number or None